pkg/controller/nodetune/sync.go

```diff
@@ -124,18 +109,84 @@ func (ncdc *Controller) sync(ctx context.Context) error {
 		return objectErr
 	}
 
-	nodeStatus := &v1alpha1.NodeConfigNodeStatus{
+	nodeStatus := &scyllav1alpha1.NodeConfigNodeStatus{
 		Name: ncdc.nodeName,
 	}
 
 	var errs []error
-
-	err = ncdc.syncJobs(ctx, jobs, nodeStatus)
+	err = controllerhelpers.RunSync(
+		&statusConditions,
+		fmt.Sprintf(jobControllerNodeTuneProgressingConditionFormat, ncdc.nodeName),
+		fmt.Sprintf(jobControllerNodeTuneDegradedConditionFormat, ncdc.nodeName),
+		nc.Generation,
+		func() ([]metav1.Condition, error) {
+			return ncdc.syncJobs(ctx, nc, jobs, nodeStatus)
+		},
+	)
 	if err != nil {
 		errs = append(errs, fmt.Errorf("can't sync jobs: %w", err))
 	}
 
-	err = ncdc.updateNodeStatus(ctx, nodeStatus)
+	// Aggregate node conditions.
+	var aggregationErrs []error
+	nodeAvailableConditionType := fmt.Sprintf(internalapi.NodeTuneAvailableConditionFormat, ncdc.nodeName)
```

Comment:
```suggestion
	nodeTuneAvailableConditionType := fmt.Sprintf(internalapi.NodeTuneAvailableConditionFormat, ncdc.nodeName)
```
Otherwise it's ambigous. The same with below ones.

Reply:
I started with this and moved to `nodeAvailableConditionType` as I figured it should be clear from context, but in that case I don't mind going back to it.